polars/polars-lazy/src/physical_plan/streaming/checks.rs

```diff
@@ -3,6 +3,10 @@ use polars_plan::prelude::*;
 
 pub(super) fn is_streamable_sort(args: &SortArguments) -> bool {
     // check if slice is positive
+    if args.maintain_order {
```

Comment:
We shouldn't print here. And can we combine the check with the slice check.

Reply:
done